docs/mobility/slam/cartographer/_index.md

```diff
@@ -18,6 +18,9 @@ aliases:
 To use Cartographer with the Viam {{< glossary_tooltip term_id="slam" text="SLAM" >}} service, you can use the [`cartographer`](https://app.viam.com/module/viam/cartographer) {{< glossary_tooltip term_id="modular-resource" text="modular resource" >}}.
 See the [Use Modules](/registry/#use-modules) section for instructions on using a module from the Viam registry on your machine.
 
+This module utilizes the Cartographer SLAM algorithm to provide tools for generating SLAM maps on your machine.
```

Comment:
```suggestion
This module utilizes the Cartographer SLAM algorithm to provide tools for generating SLAM maps.
```

Suggesting to remove "on your machine" because generating SLAM maps ("Create new map" mode and "Update existing map" mode) happens in the cloud, not on the user's machine.

("Localization only" mode does happen on the user's machine)